src/movement/rotation_engine.cpp

```diff
@@ -7,6 +7,29 @@ phi = pitch
 theta = yaw
```

Comment:
This file looks like it could use some better documentation. Perhaps an issue for a later date, though it should be at least flagged as such for now.